sdk/highlight-py/e2e/highlight_fastapi/main.py

```diff
@@ -22,6 +25,20 @@
 app.add_middleware(FastAPIMiddleware)
 
 router = APIRouter()
+r = redis.Redis(host="localhost", port=6379, decode_responses=True)
+
+aws_key = os.getenv("E2E_AWS_ACCESS_KEY")
+aws_secret = os.getenv("E2E_AWS_SECRET_KEY")
+aws_region = os.getenv("E2E_AWS_REGION", "us-east-2")
+sqs_queue_url = os.getenv("SQS_QUEUE_URL")
+
+s3 = boto.connect_s3(aws_key, aws_secret)
+sqs = boto3.client(
+    "sqs",
+    aws_access_key_id=aws_key,
+    aws_secret_access_key=aws_secret,
+    region_name=aws_region,
+)
```

Comment:
what happens if the env doesn't have the aws creds? does this example still run?
could we update the E2E readme to mention how to set the env vars to test this?